### DISCUSSION TITLE
过期：mpv.net_CM 进度条上的缩略图不能正常显示的问题

### DISCUSSION THREAD

**Discussion by ZDY321** [hooke007/mpv_PlayKit#95] [回收]
大佬你好，我是win10用户，使用的是你链接里的mpv.net_CM软件，在把 thumbnailer.conf 中“显示”和“生成缩略图”都选择“yes”后，将鼠标放在进度条上，不能正常显示。使用了绝对路径指向 ffmpeg.exe [，还是无法正常显示，能解答一下吗，感谢！
![Snipaste_2022-03-18_13-31-24](https://user-images.githubusercontent.com/81518210/158965260-d57618d4-9c7d-4616-a101-e501dae1d139.jpg)
![Snipaste_2022-03-18_13-29-47](https://user-images.githubusercontent.com/81518210/158965264-32905d21-b4e0-4e3c-aeb3-8c497a3c347a.jpg)

**Comment by hooke007** [comment]
打开控制台看报错，我猜你路径写错了

**Reply by ZDY321** [reply]
感谢，成功了。把 ffmpeg.exe的绝对路径禁掉就好了，昨天没启用时也是无法显示，今天竟然成功了，挺迷的，哈哈

**Reply by hooke007** [reply]
只要PATH里有个ffmpeg就行，不一定非要用这个选项指定

**Reply by ZDY321** [reply]
mpv.net_CM压缩包解压之后里面没有ffmpeg程序啊；我的ffmpeg是安在D盘里的，那个绝对路径感觉也没写错啊，不就是那个bin文件夹下吗。有点疑惑

**Reply by hooke007** [reply]
> mpv.net_CM压缩包解压之后里面没有ffmpeg程序啊

本来就没有

> 那个绝对路径感觉也没写错

前面讲了“打开控制台看报错”
